src/BulletHell/Powerups/Powerups.java

```diff
@@ -0,0 +1,10 @@
+package BulletHell.Powerups;
+
+import edu.macalester.graphics.GraphicsGroup;
+
+public interface Powerups {
```

Comment:
An interface should usually have a singular name: `Powerup`, not `Powerups`.

Good use of polymorphism here!